pkg/trace/operation.go

```diff
@@ -169,7 +177,7 @@ func FromContext(ctx context.Context) (Operation, error) {
 	case operation:
 		o.operation = val
 	default:
-		return Operation{}, fmt.Errorf("not an Operation")
+		return Operation{Context: context.Background()}, fmt.Errorf("not an Operation")
```

Comment:
Now that I am rereading the manual though it also says that you should not store a context in a struct :/ hmm